tests/test_client.py

```diff
@@ -0,0 +1,99 @@
+"""Test client functions"""
```

Comment:
Start python files with `#!/usr/bin/env python3` for better portability (https://blogs.gnome.org/mcatanzaro/2018/02/16/on-python-shebangs/).

Reply:
The shebang is generally meant for scripts that are executable: https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take.  

Since this is a part of a package, you are installing this package within a specific python environment already.

Reply:
Sounds good then!